src/TestFramework/PhpUnit/Coverage/CoverageXmlParser.php

```diff
@@ -153,7 +153,7 @@ private function getCoveredLinesData(\DOMNodeList $lineCoverageNodes): array
 
         foreach ($lineCoverageNodes as $lineCoverageNode) {
             /** @var \DOMNode $lineCoverageNode */
-            $lineNumber = (int) $lineCoverageNode->getAttribute('nr');
```

Comment:
`$lineNumber` is used as a key in the array `$array[$lineNumber]`. PHP automatically converts all numeric strings to integer keys so no type casting is needed